vnext/src/Libraries/Utilities/BackHandler.windows.js

```diff
@@ -48,6 +48,20 @@
 
 type BackPressEventName = 'backPress' | 'hardwareBackPress';
 
+const DEVICE_BACK_EVENT = 'hardwareBackPress';
+
+const _backListeners = [];
+
+RCTDeviceEventEmitter.addListener(DEVICE_BACK_EVENT, () => {
```

Comment:
If we're copying code from the Android variant, would you mind updating the `baseFile` for `BackHandler.windows.js` to the Android version in `vnext/src/override.json`? Basically just replace `.ios` with `.android` here:

```json
    {
      "type": "derived",
      "file": "Libraries\\Utilities\\BackHandler.windows.js",
      "baseFile": "Libraries\\Utilities\\BackHandler.ios.js",
      "baseVersion": "0.62.0-rc.3",
      "baseHash": "aa2d482b80ae66104d632c75c24646fdcbe916af",
      "issue": "LEGACY_FIXME"
    },
```

This lets our tooling merge in changes made to the Android version of this.